boxoffice/views/admin_discount.py

```diff
@@ -152,13 +154,40 @@ def admin_edit_discount_policy(discount_policy):
             errors=discount_policy_form.errors)
 
 
+def jsonify_delete_discount_policy(discount_policy_dict):
+    discount_policy = discount_policy_dict['discount_policy']
+    form = forms.Form()
+    if request.method == 'GET':
+        return jsonify(form_template=render_form(form=form, title=u"Delete discount policy", submit=u"Delete", with_chrome=False))
+
+    if not form.validate_on_submit():
+        return api_error(message=_(u"The discount policy could not be deleted."),
+            status_code=400,
+            errors=form.errors)
+
+    db.session.delete(discount_policy)
+    db.session.commit()
+    return api_success(result={}, doc="Discount policy deleted.", status_code=200)
+
+
+@app.route('/admin/o/<org_name>/discount_policy/<discount_policy_id>/delete', methods=['GET', 'POST'])
+@lastuser.requires_login
+@render_with({'text/html': 'index.html.jinja2', 'application/json': jsonify_delete_discount_policy})
```

Comment:
Is this really meant to render with `index.html.jinja2`?

Reply:
Yes, that's the boot up page for the SPA.

Reply:
A comment to that effect will help.